statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -840,7 +891,31 @@ def forecast(self, y, steps):
         -----
         Lutkepohl pp 37-38
         """
-        return forecast(y, self.coefs, self.intercept, steps)
+        if self.exog is None and exog_future is not None:
+            raise ValueError("No exog in model, so no exog_future supported "
+                             "in forecast method.")
+        if self.exog is not None and exog_future is None:
+            raise ValueError("Please provide an exog_future argument to "
+                             "the forecast method.")
+        trend_coefs = None if self.coefs_exog.size == 0 else self.coefs_exog.T
+
+        exogs = []
+        if self.trend.startswith("c"):  # constant term
+            exogs.append(np.ones(steps))
+        exog_lin_trend = np.arange(self.n_totobs + 1,
+                                   self.n_totobs + 1 + steps)
```

Comment:
I'm not sure: above the trend was adjusted/shifted by lags, Is self.n_totobs the last linear trend value in the estimation regressor?